articles/how-to-setup-ci-cd-to-vue-with-firebase-hosting-and-github-actions/index.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/ci-cd-setup-vue/
+title: How to  Set Up CI/CD in a Vue Project with Firebase Hosting and Github Actions
+description: In this tutorial, we walk through how to set up CI/CD in a Vue Project with Firebase Hosting and Github Actions.
+author: worawat-kaewsanmaung
+date: 2020-04-24T00:00:00-07:00
+topics: [Vue,CI/CD]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/ci-cd-setup-vue/hero.png
+    alt: CI CD VUE 
+---
+Most of the software development workflow includes running, testing, building, and deploying the project manually. Each time we have any changes or updates we push the code to our version control system and deploy it manually. In any development phase, it can be measured inefficient to fix bugs and update new features manually as it consumes a lot of time and accuracy. 
+
+Well, the solution to this is to automate the entire deployment process. And, this can be done by applying CI/CD to our project. And here, we are going to do that using Vue, Firebase Hosting, and GitHub workflows. The overall process automates the process of software development and management and makes it easy to test and deploy the final results. The bottom line is to automatically build and deploy the updates to the main server after each pull request or change has been merged to the master branch of our GitHub repository. Hence, the overall process involves:
```

Comment:
sounds better as:
 _This can be done by applying CI/CD to a/your project. Here, we are going to..._